app/views/cluster.scala.html

```diff
@@ -1,134 +1,134 @@
-							</tr>
-						</table>
-					</div>
-				</div>
-			</div>
-			<!-- dslink summary -->
-			<div class="col-md-8">
-				<div class="widget wblue">
-					<div class="widget-head">DSLinks</div>
-					<div class="widget-content">
-						<div class="table-responsive">
-							<table class="table table-striped table-bordered table-hover">
-								<thead>
-									<tr>
-										<th>&nbsp;</th>
-										<th>Requesters</th>
-										<th>Responders</th>
-										<th>Duals</th>
-										<th><strong>Total</strong></th>
-									</tr>
-								</thead>
-								<tbody>
-									<tr>
-										<td><strong>Connected</strong></td>
-										<td>@dslinkStats.requestersOn</td>
-										<td>@dslinkStats.respondersOn</td>
-										<td>@dslinkStats.dualsOn</td>
-										<td>@dslinkStats.totalOn</td>
-									</tr>
-									<tr>
-										<td><strong>Disconnected</strong></td>
-										<td>@dslinkStats.requestersOff</td>
-										<td>@dslinkStats.respondersOff</td>
-										<td>@dslinkStats.dualsOff</td>
-										<td>@dslinkStats.totalOff</td>
-									</tr>
-									<tr>
-										<td><strong>Total</strong></td>
-										<td>@dslinkStats.requesters</td>
-										<td>@dslinkStats.responders</td>
-										<td>@dslinkStats.duals</td>
-										<td>@dslinkStats.total</td>
-									</tr>
-								</tbody>
-							</table>
-						</div>					
-					</div>
-				</div>
-			</div>
-		</div>
-		
-		<!-- cluster nodes -->
-		<div class="row-fluid">
-			<div class="col-md-12">
-				<div class="widget wviolet">
-					<div class="widget-head">Nodes</div>
-					<div class="widget-content">
-						<div class="table-responsive">
-							<table class="table table-striped table-bordered table-hover">
-								<thead>
-									<tr>
-										<th>Leader</th>
-										<th>Address</th>
-										<th>UID</th>
-										<th>Roles</th>
-										<th>DSLinks</th>
-										<th>Uplinks</th>
-										<th>Status</th>
-									</tr>
-								</thead>
-								<tbody>
-								@for(node <- nodes.values) {
-									<tr>
-										<td>@if(node.leader){<i class="fa fa-check-circle-o fa-2x green"></i>}</td>
-										<td>@node.address</td>
-										<td>@node.uid</td>
-										<td>@node.roles.mkString(",")</td>
-										<td>@{dslinkStats.nodeStats.get(node.address).map(_.total).getOrElse(0)}</td>
-										<td>@{uplinkStats.nodeStats.get(node.address).map(_.total).getOrElse(0)}</td>
-										<td>
-										@if(node.status == "Unreachable") {
-											<span class="label label-important">@node.status</span>
-										} else {
-											<span class="label label-success">@node.status</span>
-										}									
-										</td>
-									</tr>
-								}
-								</tbody>
-							</table>
-						</div>					
-					</div>
-					<div class="widget-foot">
-						<strong>@{nodes.size}</strong> node@if(nodes.size != 1) {s} found.
-					</div>					
-				</div>
-			</div>
-		</div>
-		
-	</div>
-</div>
+    <div class="matter">
```

Comment:
We definitely should share some format config. A lot of messy 'format' lines in commits.

Reply:
Actually, i have no idea how those changes come in. Should i change it back?